testutil/runner.go

```diff
@@ -347,6 +348,9 @@ func (r *Runner) SendListMessage(channel model.Snowflake) {
 		buffer.WriteString(" - ?help: ")
 		buffer.WriteString(help.MsgHelpHelp)
 		buffer.WriteString("\n")
+		buffer.WriteString(" - ?karmalist: ")
+		buffer.WriteString(karmalist.MsgHelpKarmaList) ///lkasdflsdafjlksadfj;lkasd;lkfjas;ldfkjasd;fkjasd;fkjasd;fkjasd;fkjas;lfkjasdkflj
```

Comment:
what in tarnation is all that about? haha

Reply:
Oh, was it not clear from the error why this was failing? Or was it a surprise that it automagically added this to the list of things that were generated?